lib/registration.pm

```diff
@@ -274,7 +274,9 @@ sub fill_in_registration_data {
     # Process modules on sle 15
     if (is_sle '15+') {
         my $modules_needle = "modules-preselected-" . get_required_var('SLE_PRODUCT');
-        if (check_var('BETA', '1')) {
+        # use HDDVERSION to confirm won't check the scc-beta-filter-checkbox when hdd version is 15
+        # since 15 is already out of beta phase and no such checkout box any more
+        if (check_var('BETA', '1') && !check_var('HDDVERSION', '15')) {
```

Comment:
Do you plan to set this variable to something like 15-SP1 in future? I believe we need same thing as we have in is_sle, but for HDD_VERSION. WDYT? With this change only, I believe we'll face same issue in 6 months.

Reply:
Since it looks like version related checking on both of base system and target system, IMO could introduce version switch to online migration like what we did in offline migration. Just a brief think, you could try it out in this way to see if it works.

Reply:
I changed the code to check screen 'scc-beta-filter-checkbox' first to avoid to judge the version, please have a look.
Verified log: http://openqa-apac1.suse.de/tests/1919#step/yast2_migration/15